### DISCUSSION TITLE
Downgrade from Collaborator to Customer

### DISCUSSION THREAD

**Discussion by tudordanbalas** [onepub-dev/onepub#542] [Support] [answered]
Is it possible to downgrade some users from Collaborators to Customers?

**Comment by bsutton** [comment]
Can you tell me about your use case?

Also how do you expect the user base to grow?

**Reply by tudordanbalas** [reply]
I didn't know about the Customers Distribution License when I invited some customers. 
Some of them will remain as collaborators to have access to generate/remove the CI/CD token, but the rest of them can be converted to Customers since it's cheaper.

**Comment by bsutton** [comment]
Can you give me the list of the customers?

**Reply by tudordanbalas** [reply]
[redacted]

**Comment by bsutton** [comment, marked as answer]
Tudor,
as requested, we have moved the above users from Collaborators to customers.

As noted we haven't had to do this before so it's a little experimental.
Our testing suggests that it should all be good but please do some basic
checks to ensure that everything is working.

Any issues please let me know.

Brett
